src/main/java/jenkins/plugins/jfrog/BinaryInstaller.java

```diff
@@ -17,6 +40,16 @@
  * @author gail
  */
 public abstract class BinaryInstaller extends ToolInstaller {
+    /**
+     * decoded "[RELEASE]" for thee download url
+     */
+    private static final String RELEASE = "[RELEASE]";
```

Comment:
We can encode it here, instead of encoding it every time
```suggestion
    private static final String RELEASE = URLEncoder.encode("[RELEASE]", StandardCharsets.UTF_8);
```